src/documents/TextfieldStepperAPI.md

```diff
@@ -0,0 +1,340 @@
+---
+title: Textfield Stepper
+layout: default
+category: Components
+---
+
+The **Textfield Stepper** is user interface component for adjusting a number value using a [Textfield](Textfield.md) for direct entry of the value, along buttons for incrementing and decrementing the value. A user can also increment/decrement the values using the up/down arrow keys.
+
+## Creator ##
+
+Use the following function to create a Textfield Stepper component:
+
+<table>
+    <tbody>
+        <tr>
+            <th>Method</th>
+            <td>
+                <code>fluid.textfieldStepper(container, options);</code>
+            </td>
+        </tr>
+        <tr>
+            <th>Description</th>
+            <td>
+                Instantiates the Textfield Stepper component. Connects a a textfield and a pair of increment/decrement buttons to set a model value. Entries are restricted to numbers within a given range.
```

Comment:
"... set a value...". remove "model"? Same comment applies to docs for text field and text field slider.